### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by arpit01923**
Avatar component completed

### CODE REVIEW

File: css/components/Avatar.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.rounded-avatar,.retina-avatar{
```
Comment (bajajcodes) [inline css/components/Avatar.css]: Here I believe `avatar` class name can be used and one retina-avatar class style can be placed as default style.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+*{
+    box-sizing: border-box;
+}
+body{
+    font-size: 62.5%;
+}
```
Comment (bajajcodes) [inline css/style.css]: Add new line at end of file

For why refer this: 
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

**Review by bajajcodes — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 2 above)
Comment (arpit01923) [reply css/style.css]: Resolved

File: css/components/Avatar.css (hunk 1 above)
Comment (arpit01923) [reply css/components/Avatar.css]: resolved

**Review by maitrakhatri — APPROVED** [review]

**Review by HarshithaKonduru214 — APPROVED** [review]

**Comment by arpit01923** [comment]
image

**Pull request merged**